crates/maili/Cargo.toml

```
@@ -59,7 +59,7 @@ jsonrpsee = ["maili-rpc?/jsonrpsee"]
 # `no_std` support
 registry = ["dep:maili-registry"]
 protocol = ["dep:maili-protocol"]
-common = ["dep:maili-common"]
+common = ["dep:maili-consensus"]
```

We should probably name this feature as `consensus` now right.

```suggestion
consensus = ["dep:maili-consensus"]
```